install/kubernetes/helm/istio/values-istio-gateways.yaml

```diff
@@ -0,0 +1,325 @@
+# Common settings.
+global:
+  # Default hub for Istio images.
+  # Releases are published to docker hub under 'istio' project.
+  # Daily builds from prow are on gcr.io, and nightly builds from circle on docker.io/istionightly
+  hub: gcr.io/istio-release
+
+  # Default tag for Istio images.
+  tag: release-1.0-latest-daily
+
+  # Gateway used for legacy k8s Ingress resources. By default it is
+  # using 'istio:ingress', to match 0.8 config. It requires that
+  # ingress.enabled is set to true. You can also set it
+  # to ingressgateway, or any other gateway you define in the 'gateway'
+  # section.
+  ingressSelector: ingress
```

Comment:
Also please change everywhere to k8sIngressWorkloadLabelValue: ingress

Reply:
sure! I can open a WI to refine this.